configs/dev-genesis.js

```diff
@@ -38,10 +38,9 @@ config.express.session.prefix = 'veres-one.';
 config.mongodb.name = 'veres_one_localhost_genesis';
 config.mongodb.host = 'localhost';
 config.mongodb.port = 27017;
-config.mongodb.local.collection = 'veres_one_localhost_genesis';
-config.mongodb.username = 'veres';
-config.mongodb.password = 'password';
-config.mongodb.adminPrompt = true;
+//config.mongodb.username = 'veres';
+//config.mongodb.password = 'password';
+//config.mongodb.adminPrompt = true;
```

Comment:
I don't know why these things are commented out. Can you leave a comment or fix it?

Reply:
this is old actually. When we updated to the latest version of mongodb it turns out that trying to login to a mongodb server with out auth enabled is fatal. Hence only add username, password, and authSource if you have a valid username, password, etc. already setup.